packages/Sortable/src/components/SortableItem/SortableItem.js

```diff
@@ -53,7 +54,12 @@ const SortableItem = ({ children, index, hasNumbers, onRemove, ...moreProps }) =
           </div>
           {onRemove && (
             <div css={itemCloseStyles} data-pka-anchor="sortable.item.remove">
-              <Button.Icon onClick={handleRemove} kind="minor" size="small" a11yText={I18n.t("sortable.aria_remove")}>
+              <Button.Icon
+                onClick={handleRemove}
+                kind={Button.Kinds.MINOR}
+                size={ShirtSizes.MEDIUM}
```

Comment:
This is changed to medium size as requested in https://github.com/acl-services/paprika/issues/222